app/src/main/java/com/teamagam/gimelgimel/app/mainActivity/view/MainActivityPanel.java

```diff
@@ -175,4 +195,8 @@ public void onPageSelected(int position) {
         public void onPageScrollStateChanged(int state) {
         }
     }
+
+    public interface OnPanelOpenListener {
+        void onOpened();
```

Comment:
IMO rename to onPanelOpen(). I agree that this is already clear because it's a panel listener so no one will ask himself what was just opened, but I find it more conventional to have a listener called On**SomeNewState**Listener contain a method called on**SomeNewState**()